src/core/numerics/time_integrator/euler.hpp

```diff
@@ -0,0 +1,91 @@
+#ifndef PHARE_CORE_NUMERICS_TIME_INTEGRATOR_EULER_HPP
+#define PHARE_CORE_NUMERICS_TIME_INTEGRATOR_EULER_HPP
+
+#include "initializer/data_provider.hpp"
+#include "amr/solvers/solver_mhd_model_view.hpp"
+
+namespace PHARE::core
+{
+template<template<typename> typename FVMethod, typename MHDModel>
+class Euler
```

Comment:
We need to discuss, I find that the `Euler` , `FiniteVolumeEuler` and `EulerIntegrator` are confusing names. Maybe it's what it's going to be if no better name, but maybe we need some documentation.